### PR TITLE
Support for linux/arm64 architecture in github workflows

### LINKED ISSUE #535: Adding support for ARM64 Arch in Github workflows for cass-operator
```[tasklist]
### Tasks
- [x] Edit Push phase for linux/arm64 architecture ( we only need to build arm64 architecture not to use images for testing)
```

### PR DESCRIPTION
**Pull request by RachanKaur**
**What this PR does**: Support for linux/arm64 Architecture in Github workflows for cass-operator.

**Which issue(s) this PR fixes**:
Fixes #535 

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -54,6 +56,20 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Build system-logger arm64
```
Comment (burmanm) [inline .github/workflows/release.yml]: There's no need to add new part here. Just add it to the previous "platform:" part with comma in between.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -67,6 +83,19 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Build cass-operator arm64
```
Comment (burmanm) [inline .github/workflows/release.yml]: Same here.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -80,6 +109,19 @@ jobs:
...
+      - name: Push system-logger arm64
+        id: docker_push_system-logger
+        uses: docker/build-push-action@v3
+        with:
+          file: logger.Dockerfile
+          build-args: |
+            VERSION=${{ env.TARGET_VERSION }}
+          context: .
+          push: ${{ !env.ACT }}
+          tags: k8ssandra/system-logger:v1.16.0-dev.e22769d-20230609, ghcr.io/k8ssandra/system-logger:v1.16.0-dev.e22769d-20230609
```
Comment (burmanm) [inline .github/workflows/release.yml]: No need for another pushes either.

File: Dockerfile [hunk 4]
```diff
@@ -10,14 +12,14 @@ COPY go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY main.go main.go
+COPY cmd/main.go cmd/main.go
```
Comment (burmanm) [inline Dockerfile:15]: These changes will not work without the PR merged first (as there's no change cmd pkg).

File: .github/workflows/release.yml (hunk 1 above)
Comment (RachanKaur) [reply .github/workflows/release.yml:59]: Did that it gives error " ERROR: docker exporter does not currently support exporting manifest lists
Error: buildx failed with: ERROR: docker exporter does not currently support exporting manifest lists"

Comment (RachanKaur) [reply .github/workflows/release.yml]: Tried that it gives this error:
``ERROR: docker exporter does not currently support exporting manifest lists``

File: Dockerfile (hunk 4 above)
Comment (RachanKaur) [reply Dockerfile:15]: for now testing (draft PR) will remove `cmd`

**Pull request closed without merging**